### PR TITLE
 feat(t03): custom 10 jobs per page

### PR DESCRIPTION
**Pull request by vutiendanhvgu01**
dapper-gateDanh.surge.sh

### CODE REVIEW

File: src/containers/Home/index.tsx [hunk 1]
```diff
@@ -89,6 +101,41 @@ function Home() {
             })}
           </div>
         )}
+        <div>
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:104]: add class 'page'

File: src/containers/Home/index.tsx [hunk 2]
```diff
@@ -89,6 +101,41 @@ function Home() {
...
+          <button
+            onClick={() => {
+              changePage(-1);
+            }}
+            disabled={validCurrentPagePrevious}
+            className="page__change--darkBlue"
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:110]: i think just need 'page__change'

File: src/containers/Home/index.tsx [hunk 3]
```diff
@@ -89,6 +101,41 @@ function Home() {
...
+                {index + 1}
+              </button>
+            )
+          )}
+          <button
+            onClick={() => {
+              changePage(1);
+            }}
+            disabled={validCurrentPageNext}
+            className="page__change--darkBlue"
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:134]: i think just need 'page__change'

File: src/containers/Home/index.tsx (hunk 1 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:104]: I have already fixed

File: src/containers/Home/index.tsx (hunk 2 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:110]: I have already fixed

File: src/containers/Home/index.tsx (hunk 3 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:134]: I have already fixed

File: src/containers/Home/index.tsx [hunk 4]
```diff
@@ -89,6 +101,41 @@ function Home() {
...
+          >
+            Previous
+          </button>
+          {Array.from({ length: Math.ceil(jobs.length / jobsPerPage) }).map(
+            (_, index) => (
+              <button
+                className={
+                  currentPage === index + 1
+                    ? "btn__page btn__page-focus"
```
[the remark this answers is not in the document]
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:119]: I have already fixed

**Pull request merged**